SwiftWisdom/Core/UIKit/CellConfiguring.swift

```diff
@@ -26,12 +26,12 @@ public extension UITableView {
     }
     
     public func ip_dequeueCell<T: UITableViewCell>(indexPath: NSIndexPath, identifier: String = T.ip_identifier) -> T {
-        let cell = dequeueReusableCellWithIdentifier(T.ip_identifier, forIndexPath: indexPath) as! T
```

Comment:
The original implementation was ignoring the `identifier` argument regardless of what was specified, so if you registered your value with an identifier different from the auto-generated one, it would fail.

Ultimately, making sure that if we expose an `identifier` argument, we're actually using it